pkg/audit/audit_stripe.go

```diff
@@ -199,26 +170,26 @@ func calcPadded(size int64, blockSize int) int64 {
 	return size + int64(blockSize) - mod
 }
 
-// auditStripe gets remote segments from a pointer and runs an audit on shares
-// at a given stripe index
-// TODO(nat): maybe removed required/total here?
-func (a *Auditor) auditStripe(ctx context.Context, pointer *pb.Pointer, stripeIndex int) (badNodes []*pb.Node, err error) {
+// auditStripe downloads shares then audits the shares at a given stripe index
```

Comment:
` auditStripe downloads shares then verifies the data correctness at the given stripe`